File: Selecao I/jokenpo.cpp
/*@jokenpo: Jokenpo das tartarugas*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    
    char jog1, jog2;
    
    cin >> jog1;
    cin >> jog2;
    
    if((jog1 == 'R' && jog2 == 'S') || (jog1 == 'P' && jog2 == 'R') || (jog1 == 'S' && jog2 == 'P')){
        cout << "jog1\n";
    }else if((jog2 == 'R' && jog1 == 'S') || (jog2 == 'P' && jog1 == 'R') || (jog2 == 'S' && jog1 == 'P')){
        cout << "jog2\n";
    }else{
        cout << "empate\n";
    }
    return 0;
}

File: src/Selecao_I/Jokenpo.java
/**
 * Questão: @jokenpo: Jokenpo das tartarugas
 *
 * @author UxieGu1
 */
package Selecao_I;
import java.util.Scanner;

public class Jokenpo {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        char jog1 = scan.next().charAt(0);
        char jog2 = scan.next().charAt(0);

        if((jog1 == 'R' && jog2 == 'S') || (jog1 == 'P' && jog2 == 'R') || (jog1 == 'S' && jog2 == 'P')) {
            System.out.println("jog1");
        } else if((jog2 == 'R' && jog1 == 'S') || (jog2 == 'P' && jog1 == 'R') || (jog2 == 'S' && jog1 == 'P')) {
            System.out.println("jog2");
        } else {
            System.out.println("empate");
        }

        scan.close();
    }
}